viewer/pom.xml

```diff
@@ -296,7 +296,7 @@
                                 <param>${viewerhtml.path}/components/RequestManager.js</param>
                                 <param>${viewerhtml.path}/components/DataSelectionChecker.js</param>
                                 <!-- Common -->
-                                <param>${viewerhtml.path}/common/overrides.js</param>
```

Comment:
if we don't need this file it should be deleted from the repo

Reply:
Moved to OpenLayers 2 includes.